elixir/Dockerfile.postgres-dev

```diff
@@ -0,0 +1,17 @@
+# Builds a Postgres image with needed extensions and modules for local development
+
+# Install wal2json
+ARG POSTGRES_VERSION=15
+FROM postgres:${POSTGRES_VERSION}-alpine AS builder
+RUN apk add --no-cache \
+  build-base \
+  git \
+  llvm \
+  clang
+RUN git clone https://github.com/eulerto/wal2json.git
```

Comment:
I think it might be smart to checkout a particular tag here to make these builds as deterministic as possible.

Reply:
Maybe even better to download a source archive of a tag here instead of `git clone`. I think docker will cache this indefinitely but you'll be stuck on whatever was the tip of main on the first time you built the layer.

Reply:
Yeah good call.